packages/shared/src/hooks/feed/useFeedVotePost.ts

```diff
@@ -17,55 +20,70 @@ import AnalyticsContext from '../../contexts/AnalyticsContext';
 import { postEventName } from '../../components/utilities';
 import { AuthTriggers } from '../../lib/auth';
 
-export type UseFeedUpvotePostVariables = {
+export type UseFeedVotePostProps = {
   id: string;
   index: number;
 };
 
+export type UseFeedVotePost = {
+  onUpvote: (
+    post: Post,
+    index: number,
+    row: number,
+    column: number,
+    upvoted: boolean,
+  ) => Promise<void>;
+};
+
 const upvotePostKey = upvotePostMutationKey.toString();
 const cancelUpvotePostKey = cancelUpvotePostMutationKey.toString();
+const downvotePostKey = downvotePostMutationKey.toString();
+const cancelDownvotePostKey = cancelDownvotePostMutationKey.toString();
 const mutationHandlers = {
```

Comment:
Generic question can't we use these mutationHandlers for the PostActions as they are the same?

Reply:
Yeah, good idea 🚀

Reply:
Added in a6df656be4fe061914a3b8d919aad31553b6adaa